### PR TITLE
Implemented usage of pandoc renderer.

### PR DESCRIPTION
**Pull request by leipert**
Essentially the same as #27, but rebased onto master.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ current editor using the keymap `ctrl-shift-m`.
     ````bash
     $ apm install mathjax-wrapper
     ````
+  4. Optionally you may use pandoc to render the Markdown preview. Please have a look at the pandoc section below troubleshooting.
```
Comment (Galadirith) [inline README.md]: I removed the troubleshooting section in Galadirith/markdown-preview-plus@5d764120d19c9453fe6fd8319c40a5a37d6b02d1. Could you please remove the reference to it.

File: README.md [hunk 2]
```diff
@@ -48,6 +49,40 @@ current editor using the keymap `ctrl-shift-m`.
 Should you have any problems while installing or using MPP please open up a
 [new issue](https://github.com/Galadirith/markdown-preview-plus/issues/new).
 
+- **I want to use pandoc but it doesn't work**
```
Comment (Galadirith) [inline README.md]: Could you remove L52 and L53. They don't really have a place any more as I removed the troubleshooting section however I don't think that really matters as L43 can point them to the Pandoc section.

**Comment by leipert** [comment]
@Galadirith: Fixed README.md accordingly.

**Comment by Galadirith** [comment]
@leipert Thats awesome thanks for doing that. I've got it merged locally and I'll have it merged on GitHub shortly.

Sorry its taken longer then the weekend, while fixing the specs my `config.cson` kept getting overwritten. I've finally isolated the problem spec and everything's passing so should be publishing soon :D

**Comment by leipert** [comment]
:dancer:

**Pull request merged**